examples/src/main/scala/examples/hybrid/history/HybridHistory.scala

```diff
@@ -450,17 +465,24 @@ class HybridHistory(val storage: HistoryStorage,
     chainBack(storage.bestPosBlock, isGenesis).get.map(_._2).map(Base58.encode).mkString(",")
   }
 
-  //todo: real impl
   override def reportSemanticValidity(modifier: HybridBlock,
                                       valid: Boolean,
```

Comment:
Maybe it would be more elegant to change the type of `valid` to the superclass of `Valid` and `Invalid`...